setup.py

```diff
@@ -203,11 +203,14 @@ def get_local_version(version: "ScmVersion", time_format="%Y%m%d") -> str:
         "torch>=1.4.0",
         "tqdm",
         "scikit-learn>=0.21.2",
-        "seals>=0.1.5",
+        "seals~=0.1.5",
         STABLE_BASELINES3,
         "sacred>=0.8.4",
         "tensorboard>=1.14",
-        "huggingface_sb3>=2.2.1",
+        # TODO: remove once https://github.com/huggingface/huggingface_sb3/issues/37 is
```

Comment:
```suggestion
        # TODO(ernestum): remove once https://github.com/huggingface/huggingface_sb3/issues/37 is
```
Good to leave username or issue ID so we have something to follow up on to find more information about the reason for the TODO.